src/dc_configure.c

```diff
@@ -786,6 +812,19 @@ int dc_is_configured(const dc_context_t* context)
 }
 
 
+/*
+ * Check if there is an ongoing process.
+ */
+int dc_has_ongoing(dc_context_t* context)
```

Comment:
why "process"?  Is this True for both configuration and verification or so? 
IMHO It's better if it's a specific API about configure like "dc_configure_is_ongoing"
and "dc_verification_is_ongoing" (if needed at all).

Reply:
it's true for configure, import and export. these "processes" are mutual exclusive.

Reply:
btw. the whole ongoing-function-group is mainly for allowing the interruption of the configure-/import-/export-/key-transfer-processes (interrupting is just done by checking a flag from time to time, typically together with emitted process events). 
i liked to have _one_ function that can interrupt such things, sth. as CTRL-C :)

the check to _not_ start a process if another is still running is more a side effect - if we skip it completely, the jobs would be just enqueued, however, i do not think the user of the lib would do this intentionally, so a check makes some sense - and is rather cheap as the ongoing-stuff is already "allocated" to allow interruption.